meshtastic/mesh_interface.py

```diff
@@ -239,7 +240,7 @@ def getNode(self, nodeId, requestChannels=True):
 
     def sendText(
         self,
-        text: AnyStr,
```

Comment:
While this was marked as `AnyStr` (which is basically "either string or bytes, but not a mix of the two"), we treated it afterwards as always being a string. mypy complained about that and I think it's right to have this be a plain str instead.